background.js

```diff
@@ -0,0 +1,36 @@
+const projectHosts = [
+    'wikipedia.org',
+    'wikidata.org',
+    'wikivoyage.org',
+    'wiktionary.org',
+    'wikibooks.org',
+    'mediawiki.org',
+    'wikisource.org',
+    'wikiversity.org',
+    'wikinews.org',
+    'wikiquote.org',
+    'meta.wikimedia.org',
+    'incubator.wikimedia.org',
+    'commons.wikimedia.org',
+    'species.wikimedia.org'
+]
+
+function isSupportedHost(hostname) {
+    for (let i = 0; i < projectHosts.length; i++) {
```

Comment:
Is it safe to use let/const here?

Reply:
let/const seem well supported in modern browsers.[1]  The MediaWiki browser compatibility matrix[2] goes further back but I am not sure we need to be bound by that here.  My preference would be to target modern language constructs where reasonably widely adopted, and update for compatibility with older software if there is demand to do so.

[1] https://caniuse.com/#feat=let, https://caniuse.com/#feat=const
[2] https://www.mediawiki.org/wiki/Compatibility#Browsers

Reply:
Ok, was just checking. Looks like the only browser that don't support this are older versions of IE and Mobile IE, before 11, which I think are not much in use these days. So, I guess that is fine.